ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala

```diff
@@ -17,8 +17,12 @@ trait ProcessRepository[F[_]] extends Repository[F] with EspTables {
   import api._
   import pl.touk.nussknacker.ui.security.api.PermissionSyntax._
   protected def processTableFilteredByUser(implicit loggedUser: LoggedUser): Query[ProcessEntityFactory#ProcessEntity, ProcessEntityData, Seq] = {
+    if (loggedUser.hasAdminPermission) {
```

Comment:
let's not use explicit return and many exit points...

Reply:
Changed.